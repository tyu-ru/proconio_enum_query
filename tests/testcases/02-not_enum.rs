#[proconio_enum_query::derive_query]
struct Query1 {
    a: i64,
    b: char,
}
#[proconio_enum_query::derive_query]
struct Query2(i64, char);

fn main() {}
