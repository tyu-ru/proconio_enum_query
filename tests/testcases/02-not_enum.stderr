error: expected enum or match expression
 --> $DIR/02-not_enum.rs:1:1
  |
1 | #[proconio_enum_query::derive_query]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected enum or match expression
 --> $DIR/02-not_enum.rs:6:1
  |
6 | #[proconio_enum_query::derive_query]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
